daemon/cmd/status.go

```diff
@@ -46,6 +46,7 @@ import (
 	"github.com/cilium/cilium/pkg/option"
 	"github.com/cilium/cilium/pkg/rand"
 	"github.com/cilium/cilium/pkg/status"
+	v "github.com/cilium/cilium/pkg/version"
```

Comment:
Nit: is there a reason to do a named import here? Why not just use the package name as long as there's no conflicts already, which there shouldn't be.

Reply:
> Nit: is there a reason to do a named import here? Why not just use the package name as long as there's no conflicts already, which there shouldn't be.

Thanks for the remark.
Done. Please check